webapp/src/modules/address/reducer.js

```diff
@@ -9,15 +9,21 @@ import {
   FETCH_ADDRESS_CONTRIBUTIONS_SUCCESS,
   FETCH_ADDRESS_CONTRIBUTIONS_FAILURE
 } from './actions'
+import { buildCoordinate } from 'lib/utils'
 import { toAddressParcelIds } from './utils'
 
+const EMPTY_ADDRESS = {
+  contributions: [],
+  parcel_ids: []
+}
+
 const INITIAL_STATE = {
   data: {},
   loading: true,
   error: null
 }
-
 export default function reducer(state = INITIAL_STATE, action) {
+  const { TRANSFER_PARCEL_SUCCESS } = require('modules/transfer/actions')
```

Comment:
whops, why's this here?

Reply:
oh yea, this was a workaround due to a cyclic import. if i import that at the top of the document is undefined. any better ideas to solve this?

Reply:
Aw, at the top of my head...not really. But I think we should leave a comment explaining the situation, no?